src/lib/eslint/EslintProcessHelper.ts

```diff
@@ -0,0 +1,77 @@
+import { CLIEngine } from 'eslint';
+import * as path from 'path';
+import { CUSTOM_CONFIG } from '../../Constants';
+import { RuleResult, RuleViolation, ESMessage, ESRule, ESReport } from '../../types';
+
+
```

Comment:
Module with common code that's used by both CustomEslintEngine and BaseEslintEngine. Most of the old code remains the same.